File: Java Implementation/src/P9.java
//Special Pythagorean triplet
public class P9 {

  public static void main(String[] args) {

    boolean flag = true;
    for (int x = 1; x < 1000; x++) {
      for (int y = 1; y < 1000; y++) {
        for (int z = 1; z < 1000; z++) {
          if ((Math.pow(x,2) + Math.pow(y,2)) == (Math.pow(z,2))) {
            if ((x + y + z) == 1000) {
              System.out.println("x: " + x + " y: " + y + " z: " + z);
              System.out.println("Product: " + (x*y*z));
              break;
            }
          }
        }
      }
    }
  }
}

File: Swift Implementation/P9.swift
import Foundation
import Darwin

for a in 1 ..< 1000 {
    for b in 1 ..< 1000 {
        for c in 1 ..< 1000 {
            if (pow(Double(a),2) + pow(Double(b),2) == pow(Double(c),2)) {
                if a + b + c == 1000 {
                    print("The product is \(a*b*c)")
                }
            }
        }
    }
}